packages/create-miniapp-cli/template/fullstack/pages/404.tsx

```
@@ -0,0 +1,31 @@
+import { Typography } from 'antd'
+import React, { CSSProperties } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+
+import { EmptyView } from '@billing-connector/domains/common/components/containers/EmptyView'
```

ESLint

[error] '@billing-connector/domains/common/components/containers/EmptyView' import is restricted from being used by a pattern.

(no-restricted-imports)